latest/tests/strict_image_suite.json

```diff
@@ -0,0 +1,388 @@
+{
+  "description": "TBD",
+  "schema": {
+    "id": "schemas/strict_image.schema"
```

Comment:
Potentially, we could use `$ref` here https://github.com/json-schema-org/JSON-Schema-Test-Suite/blob/bf383b4cf2a0b83957b232d3ba5a92222f632a08/tests/draft2019-09/id.json#L4 ?

Reply:
We could use $ref, and then have a class or function elsewhere that provides the file to ref mapping, yes.

Reply:
Ah, I think I misunderstood. Not sure if adding `"$ref"` is within the intent of the suite writers, but we could certainly use it.

Reply:
Mostly inspired by https://github.com/json-schema-org/JSON-Schema-Test-Suite/blob/060caae0dd58e34af0449baec1103606a0ef4977/test-schema.json#L23 which suggests that creating a ref to a meta-schema is not fully out of scope. But I do not advocate for making this PR too complex/hard to integrated